chance/p5js/Popup-Circles/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: p5js
+author: Daniele Maselli
+author-url: https://github.com/DanieleMaselli 
+title: Popup Circles 
+tags: [P5.js, chance, repeat, random, Law of Similarity]
```

Comment:
Nearly right. Did you take a look at the link? https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#markdown-and-front-matter

We need to define the tags in a diffenret way then before. 🙇🏽‍♂️